src/ApparentHorizons/Tags.hpp

```diff
@@ -326,5 +327,22 @@ struct SurfaceIntegral : db::ComputeTag {
                                    StrahlkorperTags::Strahlkorper<Frame>>;
 };
 
+struct Area : db::SimpleTag {
```

Comment:
Add a doxygen comment and check that it renders correctly:

`/// Tag representing the surface area of a Strahlkorper`